README.md

```diff
@@ -71,6 +71,36 @@ some.trigger("hi");
 // Nothing happens
 ```
 
+### 3. Typescript Support
+You can use [Typescript Generics](https://www.typescriptlang.org/docs/handbook/generics.html) to define events that component can emit.
+```ts
+import Component from "@egjs/component";
+
+class SomeClass extends Component<{
```

Comment:
@WoodNeck 

How about splitting it by interface?
I can't recognize it.

```ts
interface ComponentEvents {
    events1;
   events2;
}
```

class SomeClass extends Component<ComponentEvents> {}

Reply:
Well... we can't in the current implementation.
![image](https://user-images.githubusercontent.com/26213435/94227025-20174680-ff34-11ea-94e8-772050e53e58.png)

Reply:
@WoodNeck 
Oh Sorry. use type instead of interface
```ts
type ComponentEvents = {
    events1;
   events2;
}
```